gradlew.bat

```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off
```

Comment:
I'd actually rather leave this one out to not imply we support building on Windows (which we don't) :D

Reply:
@passy `gradlew.bat` has been removed :)